frontend/src/pages/create_user_3.tsx

```diff
@@ -12,10 +12,16 @@ export default function CreateUser() {
   useRedirectToHomeIfSignedIn();
   const router = useRouter();
 
+  const { createSuccess } = router.query;
+  console.log("createSuccess:", createSuccess);
+
   const onBack: SubmitHandler<FieldValues> = (data) => {
     console.log(data);
     void router.push("/create_user_2");
```

Comment:
Back button should route to the login page not create user 2. Otherwise we run the risk of someone trying to create multiple accounts for themselves.

Reply:
Should we just stop displaying the "back" button if the account is created successfully? If it's not successful the user might want to go back page-by-page and check what they entered. But regardless the user wouldn't be able to make multiple accounts with the same email because Firebase would throw an error.

Reply:
No the back button should exist and just route to the login page. The main concern I had was the user submitting the create user request multiple times which could overload the backend. From a UX pov I think it makes sense for the back button to go to the login page but we could ask designers on their thoughts as well.